code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/legionnaire.dm

```diff
@@ -282,11 +301,12 @@
 
 /obj/item/crusher_trophy/legionnaire_spine
 	name = "legionnaire spine"
-	desc = "The spine of a legionnaire.  It almost feels like it's moving..."
+	desc = "The spine of a legionnaire.  With some creativity, you could use it as a crusher trophy.  Alternatively, shaking it might do something as well."
 	icon = 'icons/obj/lavaland/elite_trophies.dmi'
 	icon_state = "legionnaire_spine"
 	denied_type = /obj/item/crusher_trophy/legionnaire_spine
 	bonus_value = 20
+	var/use_time
```

Comment:
[add dmdoc to meet documentation requirements](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)